README.md

```diff
@@ -54,7 +58,7 @@ Select the **Demo** link next to **Scene and activity recognition in images** un
 Talk through example image shown by default. The ability to scroll through the JSON response and pick out the objects it found, the tags it assigned and description provided for example.
 
 ![Computer Vision Example](/docs-images/computer-vision-demo.JPG)
-
+x
```

Comment:
what is the 'x' for?

Reply:
Oops typo....I think this is an incomplete cut shortcut :-)